internal/utils.go

```diff
@@ -101,14 +104,52 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]map[string
 		)
 	}
 
-	objectTemplates := make([]map[string]interface{}, 0, len(manifests))
+	policyTemplatesLength := 1
+	if !policyConf.ConsolidatedManifests {
+		policyTemplatesLength = len(manifests)
+	}
+	objectTemplates := make([]map[string]interface{}, 0, policyTemplatesLength)
```

Comment:
I think we still want the size to be `len(manifests)` if the `policyConf.ConsolidatedManifests` is `true`.

Reply:
Correct, thanks for catching this